src/weather.cpp

```diff
@@ -57,18 +59,22 @@ void Weather::Draw(Bitmap& dst) {
 	}
 }
 
+static constexpr int num_strength = 3;
+static constexpr int num_rain_or_snow_particles[] = { 20, 60, 100 };
 static constexpr auto rain_bitmap_rect = Rect{ 0, 0, 6, 24 };
 static constexpr auto snow_bitmap_rect = Rect{ 0, 0, 2, 2 };
 static constexpr auto overlay_bitmap_rect = Rect{ 0, 0, TILE_SIZE, TILE_SIZE };
 
+static constexpr auto num_fog_particles = 2;
+
+static constexpr int num_sand_particles[] = { 128, 191, 255 };
 static constexpr int num_sand_colors = 4;
-static constexpr int num_sand_strength = 3;
 static constexpr auto sand_particle_rect = Rect{ 0, 0, 1, 2 };
 
 static constexpr auto sand_particle_bitmap_rect = Rect{
 	0, 0,
 	sand_particle_rect.width,
-	sand_particle_rect.height * num_sand_colors * num_sand_strength
+	sand_particle_rect.height * num_sand_colors,
```

Comment:
is this extra comma intentional? (I'm surprised that C++ accepts trailing commas)

Reply:
Not intentional, but modern C++ lets do you do this now.